docs/setup-and-usage.md

```diff
@@ -105,6 +105,17 @@ kubectl apply -f configs/manifests/monitoring-persist.yml
 kubectl apply -f configs/manifests/monitoring.yml
 ```
 
+* Optionally deploy the alerting manager and get alerts on Telegram.
```

Comment:
Maybe also add a link to the bot readme for how to setup. IIRC there was some additional setup right?

Reply:
I added their repo url. hopefully no additional setup required for bot to be working.